Roslyn.runsettings

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>  
+<RunSettings>  
+  <RunConfiguration>  
+    <TestAdaptersPaths>%userprofile%\.nuget\packages\xunit.runner.visualstudio\2.2.0-beta4-build1194</TestAdaptersPaths>  
```

Comment:
Why do we need this? Is there any way we can include Packages.props and use the `$(xunitVersion)` property? Lacking that it's an untracked dependency we have to update every time we change NuGet versions. That needs to be automatic for us in the future.